packages/phoenix-evals/src/phoenix/evals/evaluators.py

```diff
@@ -1183,6 +1185,7 @@ def evaluate_dataframe(
     dataframe: pd.DataFrame,
     evaluators: List[Evaluator],
     tqdm_bar_format: Optional[str] = None,
+    hide_tqdm_bar: Optional[bool] = False,
```

Comment:
```suggestion
    hide_tqdm_bar: bool = False,
```

Non-blocking: Is the optional type needed?